nucypher/characters/control/controllers.py

```diff
@@ -139,8 +140,8 @@ def test_client(self):
         try:
             from tests.utils.controllers import JSONRPCTestClient
         except ImportError:
-            raise ImportError("Nucypher is not installed in development mode,  "
-                              "Please checkout the code locally and reinstall to use the test client.")
+            raise DevelopmentInstallationRequired(importable_name='tests.utils.controllers.JSONRPCTestClient')
```

Comment:
in previous file used package name `tests.constants`, so maybe to use only package name here and in other places too?

Reply:
:heavy_check_mark: